plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -114,6 +115,19 @@ public BaseJdbcConfig setCacheMaximumSize(long cacheMaximumSize)
         return this;
     }
 
+    public boolean isReuseConnection()
+    {
+        return reuseConnection;
+    }
+
+    @Config("connection.reusing")
+    @ConfigDescription("Determines if JDBC connection are reused within single Trino query")
```

Comment:
Make it explicit this applies on coordinator only and not on workers (should there be a JDBC connector with multiple splits, or for writes, which are parallel)